### PR TITLE
added-com

### LINKED ISSUE #1: app npm commands
app npm commands

### PR DESCRIPTION
**Pull request by galaxyexp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start":"tsc && node build/app.js"
```
Comment (galaxyexp) [inline package.json:7]: change it to run

File: src/app.ts [hunk 2]
```diff
@@ -1 +1 @@
-console.log("test");
\ No newline at end of file
+console.log("more good msg");
```
Comment (jkharrs) [inline src/app.ts]: add some more good msg

Comment (galaxyexp) [reply src/app.ts]: which?

Comment (jkharrs) [reply src/app.ts]: like hello world :)

**Review by jkharrs — APPROVED** [review]

**Review by jkharrs — APPROVED** [review]

**Review by jkharrs — APPROVED** [review]

**Review by jkharrs — APPROVED** [review]

**Pull request merged**